styles/templates/_defaultElementOwl.scss

```diff
@@ -0,0 +1,182 @@
+.ontodia-default-owl-template {
+  max-width: 450px;
+  border-style: solid;
+  border-width: 2px;
+  border-color: black;
+  margin: auto;
+  max-height: 53px;
+  max-width: 170px;
+
+  &__thumbnail {
+    text-align: center;
+    > img {
+      max-width: 200px;
+    }
+  } 
+
+  &[data-expanded='true'] &__thumbnail > img {
+    max-width: 350px;
+  }
+}
+.ontodia-default-owl_thing_template {
+
+  &__thumbnail {
+    text-align: center;
+    > img {
+      max-width: 200px;
+    }
+  } 
+
+  &[data-expanded='true'] &__thumbnail > img {
+    max-width: 350px;
+  }
+}
+
+.ontodia-default-owl-template_type-line {
+  padding: 0px 7px;
+  color: white;
+  height: 18px;
+  margin-bottom: 2px;
+  overflow: hidden;
+  display: flex;
+  align-items: baseline;
+  margin-top: -3px;
+}
+
+div.ontodia-default-owl-template_type-line__icon {
+  position: inherit !important;
+  margin-right: 4px;
+}
+
+.ontodia-default-owl-template_type-line_text-container {
+  flex-grow: 1;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  width: 0;
+}
+.ontodia-default-owl-template_type-line_text-container__text {
+  display: inline;
+  white-space: nowrap;
+  font-size: 12px;
+}
+
+.ontodia-default-owl-template_body {
+  padding: 7px 10px 0px 10px;
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  &:hover
+  {
```

Comment:
Please move this brace to the previous line.

Reply:
done